src/resources/postcss/views/full/month/_day.pcss

```diff
@@ -63,16 +79,34 @@
 			opacity: 0.6;
 		}
 
+		.tribe-events-calendar-month__day-date,
+		.tribe-events-calendar-month__day-date-link {
+			color: var(--tec-color-day-marker-past-month);
+
+			&:hover,
+			&:focus {
+				color: var(--tec-color-day-marker-month-past-hover);
+			}
+
+			&:active {
+				color: var(--tec-color-day-marker-month-past-active);
+			}
+
+			.tribe-events-calendar-month__day-date-daynum {
+				color: inherit;
+			}
+		}
```

Comment:
do we need both `__day-date` and `__day-date-link` selectors? `__day-date` is not a link, so hover, focus, and active don't apply.